SAMtemplates/load_balancer/main.yaml

```diff
@@ -25,11 +25,29 @@ Parameters:
   CommitId:
     Type: String
 
+  KmsKey:
```

Comment:
dont think you need this as an import

Reply:
The unnecessary parameter has been removed.